change/@microsoft-teams-js-b80ee3ad-b996-4147-b9a9-34e6609afbb1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "promisify getFileDownloads",
```

Comment:
I think it would be helpful to include the capability, so something like "promisify files.getFileDownloads()"

Reply:
Or actually, since we offer the deprecated function for back compat, maybe it would be clearer to say "added a promisified files.getFileDownloads" or something along those lines.

Reply:
done